docker-compose.yml

```diff
@@ -10,21 +11,23 @@ services:
             - "993:993"
             - "995:995"
         depends_on:
-            - redis
+            - cache
             - mongo
-
-    redis:
-        network_mode: host
+        volumes:
+            - ./data/dbs.toml:/wildduck/config/dbs.toml
+            - ./data/api.toml:/wildduck/config/api.toml
+    cache:
```

Comment:
Not sure if `cache` is the right container name here. Redis stores stuff like sessions. What's wrong with the previous `redis`?

Reply:
If you're working locally, its likely you have redis running for other things, same as mysql and the like. Any other name will do, just to avoid conflicts. I can revert to redis and use wildduck_redis_1 as hostname in config.

Reply:
You should always edit a docker-compose to be compatible with your own setup. `redis` is just fine as a name.